crates/block-producer/src/test_utils/store.rs

```diff
@@ -202,7 +194,8 @@ impl MockStoreSuccess {
         // update accounts
         for update in block.updated_accounts() {
             locked_accounts
-                .insert(update.account_id().into(), update.final_state_commitment().into());
+                .insert(update.account_id(), update.final_state_commitment())
+                .expect("TODO: what should we do with this error?");
```

Comment:
@Mirko-von-Leipzig What should we do with this one? This is test code, so panicking might be fine? Otherwise I'll have to add a new variant to `StoreError` that is only used in tests.